### ISSUE TITLE
Some reactions are not selectable

### ISSUE THREAD

**Issue by Wikiwix** [UnigramDev/Unigram#2872]
**Describe the bug**
Using the expanded reaction panel I cannot click three of the reactions:

* peace dove
* heart
* burning heart

Every other free reaction works, if one of the three reaction is selected on another device I can see the reaction in Unigram, but I cannot deactivate it (unless I select another reaction first) either

**To Reproduce**
1. Go to some chat
2. Right click a message, expand the reactions menu
3. Click at one of the three specific reactions
4. The reactions menu closes, but no reaction is applied to the message

**Expected behavior**
The selected message has the clicked reaction

**Screenshots**

![image](https://user-images.githubusercontent.com/5161388/196351317-ca64ac29-bb8a-424b-96fb-f91fe1f8b5c4.png)

**Version Info**
 - Unigram: 9.0 (7803) X64
 - Windows: 11 21H2

**Additional context**

Device language is German

**Comment by xezrunner** [comment]
Is this related or the same as #2869?

**Comment by makisukurisu** [comment]
That's an extended version of 2869.

As far as I know - Fela said that he found the issue. So hopefully this is going to be fixed soon.

**Comment by Wikiwix** [comment]
True, guess my search for open issues was not as good as it could have been ☹️

**Comment by xezrunner** [comment]
> True, guess my search for open issues was not as good as it could have been ☹️

To be fair, I opened my issue just yesterday 😁 Since then, Fela has confirmed that he has found the issue.
I'm still curious as to what the issue was.
